docs/Getting Started/Recurring Tasks.md

```diff
@@ -39,8 +39,8 @@ The next Sunday after 25 April 2021 is on 2 May.
 Alternatively, if you have enabled addition of [[Dates#Created date|created dates]], the file will instead now look like this, showing the date that the new task was created:
 
 ```markdown
-- [ ] take out the trash 🔁 every Sunday ➕ 2023-03-10 📅 2021-05-02
-- [x] take out the trash 🔁 every Sunday 📅 2021-04-25 ✅ 2023-03-10
+- [ ] take out the trash 🔁 every Sunday ➕ 2021-04-24 📅 2021-05-02
+- [x] take out the trash 🔁 every Sunday 📅 2021-04-25 ✅ 2021-04-24
```

Comment:
This is the documentation of the "addition of created dates" feature, so I believe it should be consistent with the previous example (lines 33-34) that has completion marked in 2021